app/assets/stylesheets/foreman_theme_satellite/theme.scss

```
@@ -13,6 +13,10 @@
   }
 }
 
+#satellite-eol-banner .eol-banner-content {
+  overflow-x:scroll;
```

I'd suggest changing it like this so that the scrollbar isn't there if it doesn't have to. Other than that, it works great!

```suggestion
  overflow-x: auto;
```

---

Thanks, that makes more sense